additional_tests/exchanges_tests/abstract_authenticated_exchange_tester.py

```diff
@@ -614,18 +619,22 @@ async def inner_test_create_and_cancel_limit_orders(self, symbol=None, settlemen
             self.check_created_limit_order(limit_order, price, size, side)
             assert await self.order_in_open_orders(open_orders, limit_order, symbol=symbol)
             await self.check_can_get_order(limit_order)
-            # assert free portfolio amount is smaller than total amount
-            balance = await self.get_portfolio()
-            locked_currency = settlement_currency if side == trading_enums.TradeOrderSide.BUY else self.ORDER_CURRENCY
-            assert balance[locked_currency][trading_constants.CONFIG_PORTFOLIO_FREE] < \
-                   balance[locked_currency][trading_constants.CONFIG_PORTFOLIO_TOTAL], (
-                    f"FALSE: {balance[locked_currency][trading_constants.CONFIG_PORTFOLIO_FREE]} < {balance[locked_currency][trading_constants.CONFIG_PORTFOLIO_TOTAL]}"
-                   )
+            if self.ORDER_IMPACTS_PORTFOLIO_FREE_BALANCE:
```

Comment:
Can we add a `else` to make sure the free balance is == to total balance when this is false ?
In these tests, it's better to always test those behaviors to avoid creating blind spots when conditions are skipped

Reply:
Good point